app/src/main/java/com/imashnake/animite/features/searchbar/SearchBar.kt

```diff
@@ -168,15 +163,15 @@ fun ExpandedSearchBar(viewModel: SearchViewModel = viewModel()) {
                     keyboardController?.show()
                 },
             textStyle = TextStyle(
-                color = Text,
+                color = MaterialTheme.colorScheme.primaryContainer,
                 fontSize = 16.sp,
                 fontFamily = manropeFamily,
                 fontWeight = FontWeight.Medium
             ),
             colors = TextFieldDefaults.textFieldColors(
-                backgroundColor = NavigationBar,
-                cursorColor = Text,
-                unfocusedIndicatorColor = NavigationBar,
+                backgroundColor = MaterialTheme.colorScheme.onPrimaryContainer,
```

Comment:
Are these intentionally locked to `onPrimaryContainer`? Assuming they are displayed on something with a background `primaryContainer`, then you should be about to use `LocalContentColor.current`. This way, changing the surface color would automatically propagate the correct color to components

Reply:
I was trying to copy the colors that the Clock app used :kappa:. Fixed it now.